app/src/main/res/layout/content_item.xml

```diff
@@ -12,20 +12,23 @@
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_margin="4dp"
-    android:paddingStart="8dp"
-    android:paddingEnd="8dp">
+    android:layout_marginStart="20dp"
```

Comment:
Nit suggestion : This spacing please confirm with @mschanteltc.
It's 18dp in mock which you converted to multiple of 4 and used 20dp . As its a chat mode i suggest we need to reduce it to 16dp.

Reply:
If I give 16dp it looks very close to the edge. So made it 20dp.